messaging/src/test/java/org/jboss/as/messaging/test/MessagingTestOperationContext.java

```diff
@@ -0,0 +1,1357 @@
+/*
```

Comment:
This fix does not require such a huge mock OperationContext. You can remove it (and the tests)